editor/animation_bezier_editor.cpp

```diff
@@ -1116,7 +1551,7 @@ void AnimationBezierTrackEdit::duplicate_selection() {
 			continue;
 		}
 
-		selection.insert(existing_idx);
+		selection.insert(Pair<int, int>(track, existing_idx));
```

Comment:
`Pair<int, int>` is used a lot throughout this class, it might be relevant to add a local typedef to make it more readable, like `IntPair`?